perllib/Open311/PopulateServiceList.pm

```diff
@@ -276,7 +276,7 @@ sub _set_contact_group {
         }
     }
 
-    if ($old_group ne $new_group) {
+    if ($self->_groups_different($old_group, $new_group)) {
         if ($new_group) {
             $contact->set_extra_metadata(group => $new_group);
```

Comment:
Should this always store arrayref to match admin.

Reply:
Or I guess you change it back in admin so that if one group it's always a string, not a ref of one element. I thought 'always an arrayref' would be easier, but given my other comment below, perhaps it is not.